packages/app/ui/text/BodyText.tsx

```diff
@@ -0,0 +1,40 @@
+import { TextProps } from "@/ui/text/base";
+import { cva } from "class-variance-authority";
+
+const bodyTextStyles = cva([], {
+  variants: {
+    size: {
+      3: ["text-md"],
+      2: ["text-sm"],
+      1: ["text-xs"],
+    },
+    weight: {
+      bold: ["font-bold"],
+      semibold: ["font-semibold"],
+      medium: ["font-medium"],
+    },
```

Comment:
I see we are repeating this several times, should we have a style for weights only? We could then merge both styles with `cx`

Reply:
I'll explore this idea and get back to you.

Reply:
refactored here: [27a9ce0](https://github.com/SwaprHQ/stackly-ui/pull/16/commits/27a9ce0dc09ff83ba0f3280f78e2eacf9338c1a7)